SU-HSDO/stanford_mrc
# MRCD8-218 added visual indicators for meatballs and arrows

# READY FOR REVIEW

# Summary
- Added visual indicators for focus on meatballs below slideshow and to arrows that overlay images

# Needed By (Date)
- End of sprint (3/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the focus state for the meatballs and the previous and next buttons now have focus states (leveraging the Decanter $focus-shadow variable)

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-218

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
